packages/calcite-components-react/support/patchSSR.ts

```diff
@@ -0,0 +1,32 @@
+// patch needed due to Stencil executing client side code on the server
+// when using the includeImportCustomElements option
+// https://stenciljs.com/docs/react#includeimportcustomelements
+// https://github.com/Esri/calcite-design-system/issues/7486
+
+const {
+  promises: { readFile, writeFile },
+} = require("fs");
+const { dirname, resolve } = require("path");
+
+(async () => {
+  try {
+    const filePath = resolve(`${__dirname}/../src/components.ts`);
+    const contents = await readFile(filePath, { encoding: "utf8" });
+
+    const defineCustomElementImports = /import { defineCustomElement as defineCalcite.*(\r\n|\r|\n)/gm;
```

Comment:
Can you add a short explainer on what these lines are doing (e.g., prevent static imports of `dist` output target by making CE definition lazy to accommodate SSR environments)?